pkg/config/config.go

```diff
@@ -79,6 +79,10 @@ type Config struct {
 	KeyFilePath                string             `json:"key_file" yaml:"key_file"`
 	LogLevel                   string             `json:"log_level" yaml:"log_level"`
 	LogOutput                  string             `json:"log_output" yaml:"log_output"`
+	RotateOutputPath           string             `json:"rotate_output_path" yaml:"rotate_output_path"`
+	RotationMaxSize            int                `json:"rotation_max_size" yaml:"rotation_max_size"`
+	RotationMaxAge             int                `json:"rotation_max_age" yaml:"rotation_max_age"`
+	RotationMaxBackups         int                `json:"rotation_max_backups" yaml:"rotation_max_backups"`
```

Comment:
I suggest adding a LOG prefix to make it clear that it is related to log.

e.g. `LogRotationMaxSize`

Reply:
I submit the commit to change the name, but there's dead link so i failed my test.How can i skip these checks?

Reply:
let me fix it.

Reply:
#1239 has been merge. the dead link has been fixed.

Reply:
I got it. I've merged and pushed